app.js

```diff
@@ -46,6 +46,11 @@ fs.existsSync('logs') || fs.mkdirSync('logs');
 
 const serviceBaseUrl = process.env.APPLICATION_BASE_URL || '/assessment/';
 
+
+app.get(process.env.API_DOC_URL, function (req, res) {
```

Comment:
@borkarsaish65 if this a new env variable pleasae update envVariables.js and env sample file